docs/get-started/index.md

```diff
@@ -6,3 +6,24 @@ title: Get started
 Ministry of Justice builds on the principles and guidelines published by GDS. The guidance here is intended to supplement that published by GDS and not replace it.
 
 The guidelines published here show how these have been adapted and refined to fit the goals and context of MOJ Digital & Technology.
+
+  <div class="app-main-wrapper govuk-main-wrapper govuk-main-wrapper--l">
```

Comment:
This _shouldn't_ be needed (though I haven't checked locally): the content is already wrapped in a `govuk-main-wrapper` div.

Reply:
Removed